app/screens/home/channel_list/servers/servers_list/server_item/websocket/index.ts

```diff
@@ -5,14 +5,14 @@ import withObservables from '@nozbe/with-observables';
 import {of as of$} from 'rxjs';
 import {switchMap} from 'rxjs/operators';
 
-import {observeWebsocket} from '@queries/servers/system';
+import {observeWebsocketLastDisconnected} from '@queries/servers/system';
 
 import WebSocket from './websocket';
 
 import type {WithDatabaseArgs} from '@typings/database/database';
 
 const enhanced = withObservables([], ({database}: WithDatabaseArgs) => ({
-    isConnected: observeWebsocket(database).pipe(
+    isConnected: observeWebsocketLastDisconnected(database).pipe(
```

Comment:
@enahum The way this is handled as of today, has a corner case where this does not work (therefore the observables I created). The problem here is that if the websocket has never connected, the lastDisconnected remains 0, making this thing it is connected.

Also the naming is wrong here, but it is used right (it shows an alert when `!isConnected`). If you want I can update this on this PR to use the websocket manager observable, and fix the naming.

Reply:
I wouldn't be oppose to changing it in this PR :)